app/models/rbac.rb

```diff
@@ -185,11 +185,7 @@ def self.compute_total_count(klass, scope, extra_target_ids, conditions, include
   end
 
   def self.find_reflection(klass, association_to_match)
-    klass.reflections.each do |association, reflection|
-      next unless association == association_to_match
-      return reflection
-    end
-    nil
+    klass.reflections.detect { |association, reflection| association == association_to_match }
```

Comment:
and this is why I didn't change this the first time around :(

also of note, this is called with a symbol, but all the keys in reflections are strings.

curious why we're not using `klass.reflections[association_to_match.to_s]`